tests/Unit/ParallelAlgorithms/Initialization/Actions/Test_RemoveOptionsAndTerminatePhase.cpp

```diff
@@ -24,6 +23,11 @@
 namespace {
 struct TemporalId {};
 
+struct InitialTime : db::SimpleTag {
```

Comment:
How is this related? Please remove it or put it in a separate commit. "Move" commits should literally only do the move.

Reply:
Since I’m moving this out of `Evolution` I remove the dependence on evolution tags. Are you sure you prefer this in a separate commit?

Reply:
Yes, always. "Move" commits should be completely trivial to review, otherwise it's not a move, you're actually changing code.

Reply:
ok, did the split